app/src/hooks/usePpg.ts

```diff
@@ -100,36 +105,80 @@ const usePgp = (): Hooks => {
       });
   };
 
-  const verifyPublicKey = async (email: string | undefined) => {
-    if (!email) {
-      Alert.alert(AlertTitle.Error, `No email claim found`);
-      return;
-    }
-    api
-      .getUser(email)
-      .then(async (result: UsersResponse) => {
-        if (result.emailVerified) {
-          await updateClaim(
-            ClaimTypeConstants.EmailCredential,
-            { email: email },
-            [],
-            true
+  const resendVerificationEmail = async (email: string) => {
+    const formattedEmail = email.trim().toLowerCase();
+    await api
+      .resendVerificationEmail({
+        hashedEmail: ethers.utils.hashMessage(formattedEmail)
+      })
+      .then((response) => {
```

Comment:
could be `then((response) => response && Alert.alert(etc..))`

Reply:
i think it's more clear this way, though i remove `return` cuz its sounds like we're returning some result